src/main/java/io/appium/java_client/remote/MobileOptions.java

```diff
@@ -0,0 +1,483 @@
+     * @see MobileCapabilityType#EVENT_TIMINGS
+     */
+    public T setEventTimings(boolean bool) {
+        return amend(MobileCapabilityType.EVENT_TIMINGS, bool);
+    }
+
+    /**
+     * Get whether the app reports the timings for various Appium-internal events.
+     *
+     * @return true if the app reports event timings.
+     * @see MobileCapabilityType#EVENT_TIMINGS
+     */
+    public boolean doesEventTimings() {
+        return (boolean) getCapability(MobileCapabilityType.EVENT_TIMINGS);
+    }
+
+    /**
+     * Set the app to do a full reset.
+     *
+     * @return this MobileOptions, for chaining.
+     * @see MobileCapabilityType#FULL_RESET
+     */
+    public T setFullReset() {
+        return setFullReset(true);
+    }
+
+    /**
+     * Set whether the app does a full reset.
+     *
+     * @param bool is whether the app does a full reset.
+     * @return this MobileOptions, for chaining.
+     * @see MobileCapabilityType#FULL_RESET
+     */
+    public T setFullReset(boolean bool) {
+        return amend(MobileCapabilityType.FULL_RESET, bool);
+    }
+
+    /**
+     * Get whether the app does a full reset.
+     *
+     * @return true if the app does a full reset.
+     * @see MobileCapabilityType#FULL_RESET
+     */
+    public boolean doesFullReset() {
+        return (boolean) getCapability(MobileCapabilityType.FULL_RESET);
+    }
+
+    /**
+     * Set language abbreviation for use in session.
+     *
+     * @param language is the language abbreviation.
+     * @return this MobileOptions, for chaining.
+     * @see MobileCapabilityType#LANGUAGE
+     */
+    public T setLanguage(String language) {
+        return amend(MobileCapabilityType.LANGUAGE, language);
+    }
+
+    /**
+     * Get language abbreviation for use in session.
+     *
+     * @return String representing the language abbreviation.
+     * @see MobileCapabilityType#LANGUAGE
+     */
+    public String getLanguage() {
+        return (String) getCapability(MobileCapabilityType.LANGUAGE);
+    }
+
+    /**
+     * Set locale abbreviation for use in session.
+     *
+     * @param locale is the locale abbreviation.
+     * @return this MobileOptions, for chaining.
+     * @see MobileCapabilityType#LOCALE
+     */
+    public T setLocale(String locale) {
+        return amend(MobileCapabilityType.LOCALE, locale);
+    }
+
+    /**
+     * Get locale abbreviation for use in session.
+     *
+     * @return String representing the locale abbreviation.
+     * @see MobileCapabilityType#LOCALE
+     */
+    public String getLocale() {
+        return (String) getCapability(MobileCapabilityType.LOCALE);
+    }
+
+    /**
+     * Set the timeout for new commands.
+     *
+     * @param seconds is the number of seconds to timeout before seeing a new command.
+     * @return this MobileOptions, for chaining.
+     * @see MobileCapabilityType#NEW_COMMAND_TIMEOUT
+     */
+    public T setNewCommandTimeout(int seconds) {
```

Comment:
Please use Duration type for for time units

Reply:
59d0ef99e17548c614f221e648db196aadeea623